### ISSUE TITLE
xref-backend-identifier-at-point returns a symbol but xref expects a string

### ISSUE THREAD

**Issue by rksm** [jorgenschaefer/elpy#1255]
This causes issues, e.g. one way to reproduce:

1. Open python source
2. Move point to an identifier
3. Run xref-find-references
4. Select default (M-n)

Error is thrown: `let: Wrong type argument: char-or-string-p, num_val`

Solution: Have `elpy-xref--identifier-at-point` return a string.

E.g.

```elisp
(defun elpy-xref--identifier-at-point ()
    "Return identifier at point."
    (symbol-name (symbol-at-point)))
```

**Comment by galaunay** [comment]
Thank you for the report, 
I wasn't able to reproduce the error, but you are right, xref documentation specify that the return value should be a string.
 
I made a PR #1256 to fix this.

**Comment by galaunay** [comment]
Fixed by #1277.
Closing this.
